lib/core-manager/index.js

```diff
@@ -285,6 +291,7 @@ export class CoreManager extends TypedEmitter {
         'Passed an existing protocol stream to coreManager.replicate(). Other corestores and core managers replicated to this stream will no longer automatically inject shared cores into the stream'
       )
     }
+    // @ts-ignore - too complex to type right now
```

Comment:
I generally like to use `@ts-expect-error` for these kinds of these things, in case they ever do get fixed
```suggestion
    // @ts-expect-error - too complex to type right now
```